webclient/http1/src/main/java/io/helidon/webclient/http1/Http1ClientRequestImpl.java

```diff
@@ -153,8 +153,8 @@ public UpgradeResponse upgrade(String protocol) {
         }
 
         if (response.status() == Status.SWITCHING_PROTOCOLS_101) {
-            // yep, this is the response we want
-            if (response.headers().contains(requestedUpgrade)) {
+            // is the upgrade request successful?
+            if (isUpgradeSuccessful(requestedUpgrade, response.headers().get(HeaderNames.UPGRADE))) {
```

Comment:
I believe our [dev guidelines](https://github.com/helidon-io/helidon/blob/main/DEV-GUIDELINES.md#getters-and-setters) would say that this method should be, simply, `upgradeSuccessful` since `is` is a prefix of last resort.

Reply:
Roger